src/commcare_cloud/commands/ansible/helpers.py

```diff
@@ -171,6 +171,8 @@ def get_celery_workers(environment):
 def get_celery_worker_name(environment, comma_separated_queue_name, worker_num):
     environment_environment = environment.meta_config.deploy_env
     project = environment.fab_settings_config.project
+    if comma_separated_queue_name == 'beat':
+        return "{project}-{environment}-celerybeat".format(project=project, environment=environment_environment)
```

Comment:
why is `celerybeat` this better than `celery_beat` (which is what it would be without this if statement)?

Reply:
The name is defined as `celerybeat` herre: https://github.com/dimagi/commcare-cloud/blob/master/src/commcare_cloud/ansible/roles/commcarehq/templates/supervisor_celery_beat.conf.j2#L1

And modifying it there would require modifying it on all envs, so I was going with this as a simpler solution.